genlibTest/testreadwritehelpers.cpp

```diff
@@ -49,54 +48,6 @@ TEST_CASE("vector reading and writing")
 
 }
 
-template<typename T> void comparePvecAndStdVec( const pvector<T> &pvec, const std::vector<T> &stdVec )
```

Comment:
I wonder whether we should keep these tests but move them to mgraph440?

Reply:
Moved to a new test app mgraph440Test